ocpmodels/common/relaxation/optimizers/lbfgs_torch.py

```diff
@@ -123,11 +123,14 @@ def run(self, fmax, steps):
             converged = torch.all(torch.logical_not(update_mask))
             # GPU memory usage as per nvidia-smi seems to gradually build up as
             # batches are processed. This releases unoccupied cached memory.
-            torch.cuda.empty_cache()
+        torch.cuda.empty_cache()
```

Comment:
Could you left-indent the comment above this line as well please.

Reply:
Will do.